front/package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "@auth0/auth0-react": "^2.2.1",
     "@babel/runtime": "^7.20.13",
-    "@boxtribute/shared-components": "1.0.0",
+    "@boxtribute/shared-components": "link:../shared-components",
```

Comment:
I'm not actually clear on the pros/cons to either when the component lives in the same workspace, and we're not versioning the component independently. Thoughts welcome. Very happy to switch up, both worked fine in terms of local development experience as far as I could see.

Reply:
I've switched to workspace:* , as at least we don't need a hard-coded path :) If you do know any other differences, let me know though for my own education!

Reply:
@jamescrowley Basically, `pnpm-workspace.yaml` manages the workspace packages. The `"workspace:*"` is converted to a semver representation if we publish to NPM. Otherwise, it will automatically resolve to our local package as a nice side-effect.

Reference docs: https://pnpm.io/workspaces